py-polars/polars/internals/series/series.py

```diff
@@ -3733,7 +3733,12 @@ def interpolate(self) -> Series:
         """
 
     def abs(self) -> Series:
-        """Compute absolute values."""
+        """Compute absolute values.
+
+        Same as `abs(series)`.
+        """
+
+    __abs__ = call_expr(abs)
```

Comment:
Wouldn't the following work?

```suggestion
    def __abs__(self) -> Series:
        return self.abs()
```

Reply:
> Wouldn't the following work?

At the price of an extra function call – yes.  I see that elsewhere code is duplicated in aliases to avoid run-time overhead.

Reply:
Assigning a dunder like that directly is asking for trouble. That extra function all won't hurt anyone. Maybe this will even work:

```python
    def __abs__(self) -> Series:
        pass
```

Reply:
> Assigning a dunder like that directly is asking for trouble. That extra function all won't hurt anyone. Maybe this will even work:
> 
> ```python
>     def __abs__(self) -> Series:
>         pass
> ```

It could be made to, but I think I disallowed the `dispatch` class-decorator from peering into 'private' methods by default, as a minor optimization; can revisit that decision.